### DISCUSSION TITLE
Please provide other means of donation

### DISCUSSION THREAD

**Discussion by rp2019433** [Blackymas/NSPanel_HA_Blueprint#2181] [Ideas]
Please provide other ways of making a donation for the project besides Paypal. One suggestion: Buy Me a Coffee platform.

**Comment by edwardtfn** [comment]
You can always [buy me an ice cream](https://www.buymeacoffee.com/edwardfirmo). 😉

**Comment by rp2019433** [comment]
Thank you for sharing the link. Maybe you could consider inserting it on the main page of the project with Paypal logo.

**Reply by edwardtfn** [reply]
Well, this is a @Blackymas project and that link goes to him. I'm happy to contribute here with my knowledge and time for free, so I'm not gonna change that link, although I love when someone pay me an ice cream. 😉
Thanks a lot for you donation. 🙂🍦
